src/Common/precompiled.h

```diff
@@ -210,11 +220,26 @@ typedef union _LARGE_INTEGER {
 
 // macros
 #if BOOST_OS_WINDOWS
-#define DLLEXPORT __declspec(dllexport)
+    #define DLLEXPORT __declspec(dllexport)
+    #define DLLIMPORT  __declspec(dllimport)
+    #define DEBUG_BREAK __debugbreak()
+    #define ALIGN(N) __declspec(align(N))
+    #define NOINLINE __declspec(noinline)
+    #define ASSUME(X) __assume((X)
+    #define THREAD_LOCAL __declspec(thread)
+    #define POPCNT(X) __popcnt((X))
 #else
-#define DLLEXPORT __declspec(dllexport)
+    #define DLLEXPORT
+    #define DLLIMPORT
+    #define DEBUG_BREAK
+    #define ALIGN(N) __attribute__((aligned (N)))
+    #define NOINLINE __attribute__((noinline))
+    #define ASSUME(X)
```

Comment:
Stackoverflow suggests a [solution](https://stackoverflow.com/questions/25667901/assume-clause-in-gcc/26195434#26195434), I don't know how affective it is.

Reply:
I added it in a separate PR

Reply:
See https://github.com/tomlally/Cemu/commit/73a8e6faaca15a80ae3e90cf6cbf603d9e6f03b1